entries/b/bubble.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Bubble": {
+    "domain": "bubble.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://bubble.io/support-article/what-is-2fa",
+    "keywords": [
+      "other"
```

```suggestion
      "hosting"
```

I think that this fits better in the "Hosting/VPS" category. It looks similar to webflow and carrd.